spec/controllers/facility_accounts_controller_spec.rb

```diff
@@ -79,17 +79,19 @@
       end
     end
 
-    context "when the multi_facility_accounts feature is turned off", feature_setting: { multi_facility_accounts: false, reload_routes: true } do
-      let(:purchase_order) { FactoryBot.create(:purchase_order_account, :with_account_owner, facility: facility) }
+    if EngineManager.engine_loaded?(:c2po)
+      context "when the multi_facility_accounts feature is turned off", feature_setting: { multi_facility_accounts: false, reload_routes: true } do
+        let(:purchase_order) { FactoryBot.create(:purchase_order_account, :with_account_owner, facility: facility) }
```

Comment:
I think I see that this spec is failing when the engine isn't loaded because it's referencing a PO. Is it possible to have it reference a plain account? I don't love using the status of engines to decide what specs to run. But it's not the end of the world.

Reply:
The multi-facility feature is only for POs and credit cards, so I don’t think we can use just a regular account here… Perhaps the spec belongs in the engine’s set of specs!